config/jobs/kubernetes/sig-aws/eks/periodics.yaml

```diff
@@ -0,0 +1,106 @@
+
+
+periodics:
+# Run Kubernetes 1.10 branch e2e tests with EKS prod build 1.10
+- interval: 2h
+  name: ci-kubernetes-e2e-1-10-aws-eks-1-10-prod
+  labels:
+    preset-service-account: "true"
+    preset-eks-common: "true"
+    preset-kubernetes-e2e-aws-eks-1-10: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+      imagePullPolicy: Always
+      args:
+      - --timeout=200
+      - --bare
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-version-skew=false
+      - --deployment=eks
+      - --provider=eks
+      - --gce-ssh=
```

Comment:
It's weird that this flag needs to be set for an aws job.
@BenTheElder - wdyt?

Reply:
not sure why, there's a bunch of junk in that script imho. I'm avoiding it going forward